src/Heisenslaught/Hubs/DraftHub.cs

```diff
@@ -1,213 +1,82 @@
-﻿using Heisenslaught.Infrastructure;
-using Microsoft.AspNetCore.SignalR;
+﻿using Microsoft.AspNetCore.SignalR;
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading;
 using System.Threading.Tasks;
-using Newtonsoft.Json;
+
+using Heisenslaught.DataTransfer;
+using Heisenslaught.Exceptions;
+using Heisenslaught.Services;
+
 
 namespace Heisenslaught
 {
     public class DraftHub : Hub
     {
-        public static List<string> ConnectedUsers;
-        public static Draft CurrentDraft;
-
-        private static Timer DraftUpdate;
-        private static AdminDraftConfig CurrentDraftConfig;
-        private static DraftHandler HandleDraft;
-
-    
-
-        public AdminDraftConfig ConfigDraft(DraftConfig cfg)
-        {
-            stopDraftUpdates();
-            CurrentDraftConfig = new AdminDraftConfig(cfg);
-            Clients.All.updateConfig(cfg);
-            return CurrentDraftConfig;
-        }
+        private static DraftService Draft = new DraftService();
 
-        public AdminDraftConfig getCurrentAdminConfig()
+        
+        public override Task OnDisconnected(bool stopCalled)
```

Comment:
After reading, the server will trigger `OnDisconnect` after a timeout period. I have created a follow up ticket to fix this: https://github.com/chetjan/heisenslaught/issues/29 .